### ISSUE TITLE
[FEATURE] Should have is_rewarded_video_loaded loaded

### ISSUE THREAD

**Issue by kyoz** [Poing-Studios/Godot-AdMob-Android-iOS#81]
**Describe the solution you'd like**

There should be an api to check if there is reward video loaded. Because sometime reward videos doesn't load new video after user watch first one.

With this api, we can easily detect if there's not reward video and let user know.

**Comment by gumaciel** [comment]
Hi, I think there's already some variables that you can call directly from GDScript, such as: MobileAds.interstitial_loaded

![IMG_20210621_083039_035.jpg](https://user-images.githubusercontent.com/20030153/122755158-f85ad200-d26a-11eb-91a7-bce59ff554c2.jpg)

But maybe we can do this, because will have more "security" avoiding to the programmer change the value of this variable, which would not be allowed.

**Comment by kyoz** [comment]
Yeah i just look through it. Everything working fine except the banner size is not fullwidth but floating in middle buttom, no matter what banner size i'v choosed.

 Also the reward is not auto fill, do i need to call load_reward everytime a reward video finish showing.

Thank you very much 😄
